css/new-styles.css

```diff
@@ -1,14 +1,49 @@
 .lesson-list-parent {
-    font-family: "Bitter","Open Sans",SansSerif;
+    font-family: "Bitter", "Open Sans", SansSerif;
```

Comment:
SansSerif it needs to be written like that sans-serif, therefore your line of code will look like this font-family: "Bitter", "Open Sans", sans-serif;